test/unit/routes/utils/regex-checker.test.js

```diff
@@ -2,55 +2,43 @@
 const regexChecker = require('../../../../app/routes/utils/regex-checker')
 
 describe('regexChecker', () => {
```

Comment:
Do we need a regex checker? I thought that was built into JS.

Reply:
It's a kind of misleadingly named function, it's more a 'run regex safely' method but personally I'd just get rid of it, use the regular expressions inline where they are needed and make sure non of them are invalid. I actually have a note for this a small PR for later.

Reply:
Yeah I think I prefer that too. Cool, if its in another PR then we can do it there.